src/sentry/monitors/validators.py

```diff
@@ -41,6 +42,18 @@ def to_internal_value(self, data):
         return data
 
 
+class MonitorAlertRuleTargetValidator(serializers.Serializer):
+    target_identifier = serializers.IntegerField(help_text="ID of target object")
+    target_type = serializers.CharField(help_text="One of [Member, Team]")
+
+
+class MonitorAlertRuleValidator(serializers.Serializer):
+    targets = MonitorAlertRuleTargetValidator(
+        many=True,
+        help_text="Array of dictionaries with target_identifier and target_type keys",
```

Comment:
I think it would be better to describe what `targets` is, remember the audience for this help text is people reading the documentation, so I would probably write something like

> Targets describes a list the members or teams of the organization who will be notified when a cron job fails